packages/contracts-bedrock/test/L1/SuperchainConfig.t.sol

```diff
@@ -157,47 +134,59 @@ contract SuperchainConfig_AddChain_Test is CommonTest {
         vm.expectCall(_target, _calldata);
     }
 
-    /// @notice Tests that `addChain` reverts when called by an unauthorized address.
-    function test_addChain_unauthorized_reverts(address _caller, uint256 _chainId, address _systemConfig) external {
-        vm.assume(_caller != superchainConfig.upgrader());
+    /// @notice Tests that `addDependency` reverts when called by an unauthorized address.
+    function test_addDependency_unauthorized_reverts(
+        address _caller,
+        uint256 _chainId,
+        address _systemConfig
+    )
+        external
+    {
+        vm.assume(_caller != superchainConfig.dependencyManager());
 
         vm.expectRevert(Unauthorized.selector);
         vm.prank(_caller);
-        superchainConfig.addChain(_chainId, _systemConfig);
+        superchainConfig.addDependency(_chainId, _systemConfig);
     }
 
-    /// @notice Tests that `addChain` reverts when the input chain already contains dependencies on its set.
-    function test_addChain_alreadyHasDependencies_reverts(uint256 _chainId, address _systemConfig) external {
-        // Mock the number of dependencies to be greater than 0.
-        uint256 numberOfDependencies = 1;
-        _mockAndExpect(
-            _systemConfig,
-            abi.encodeWithSelector(ISystemConfigInterop.dependencyCounter.selector),
-            abi.encode(numberOfDependencies)
-        );
+    /// @notice Tests that `addDependency` reverts when the chain is already in the dependency set.
+    function test_addDependency_chainAlreadyExists_reverts(uint256 _chainId) external {
```

Comment:
NIT: For the code order, this should go below `test_addDependency_sameChainID_reverts`